File: code/28096.pas
uses math;
program Bai0962;
var a,b,c,x,y,z:integer;
begin
  readln(a,b,c);
  x := min(a, min(b,c));
  z := max(a, max(b,c));
  y := a + b + c - x - z;
  write(x, y, z);
end.

File: code/28096.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() 
{
    int a,b,c;
    cin>>a>>b>>c;
    int x=min(a, min(b,c));
    int z=max(a, max(b,c));
    int y=a+b+c-x-z;
    cout<<x<<" "<<y<<" "<<z;
    return 0;
}
